rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/DefendantJsonMapper.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.cmc.rpa.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.models.otherparty.OrganisationDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.SoleTraderDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.domain.models.party.HasContactPerson;
+import uk.gov.hmcts.cmc.rpa.mapper.json.NullAwareJsonObjectBuilder;
+
+import java.util.List;
+import javax.json.JsonArray;
+import javax.json.stream.JsonCollectors;
+
+import static uk.gov.hmcts.cmc.rpa.mapper.helper.Extractor.extractOptionalFromSubclass;
+
+@Component
+public class DefendantJsonMapper {
+
+    @Autowired
+    private final AddressJsonMapper mapAddress;
+
+    public DefendantJsonMapper(AddressJsonMapper mapAddress) {
+        this.mapAddress = mapAddress;
+    }
+
+    public JsonArray mapDefendants(List<TheirDetails> defendants) {
```

Comment:
Please rename to simple `map`. Please to the same in all mapper classes since now there is only one map method per class. No need to repeat what is being mapped in the method name.